Mathlib/Order/Defs/PartialOrder.lean

```diff
@@ -104,6 +104,19 @@ def decidableLTOfDecidableLE [DecidableRel (α := α) (· ≤ ·)] : DecidableRe
       else isTrue <| lt_of_le_not_le hab hba
     else isFalse fun hab' => hab (le_of_lt hab')
 
+namespace Decidable
+
+variable [DecidableRel (α := α) (· ≤ ·)]
+
+lemma not_lt_iff_not_le_or_ge : ¬a < b ↔ ¬a ≤ b ∨ b ≤ a := by
```

Comment:
I don't really think we need a decidable version of this.

Reply:
:+1: (This isn't an argument for having a decidable version of this, but I'm curious why it wouldn't be a good idea for this case. Is it because it introduces a new namespace when no other lemmas that require decidability exist?)